CONTRIBUTING.md

```diff
@@ -186,8 +186,19 @@ Stable components MUST have their major version set to `1` or greater.
 
 ### Unmaintained
 
-A component which does not have an assigned component owner, or has a component owner who has been unresponsive to issues and pull requests may be marked as unmaintained.
-Unmaintained components may continue to work and receive updates and fixes from contributors, but may not receive immediate attention if there is a problem or feature request.
+A component which does not have an assigned component owner, or has a component owner who has been unresponsive to issues
+and pull requests may be marked as `pkg-status:unmaintained`.
+
+Unmaintained components may continue to work and receive updates and fixes from contributors. An unmaintained component
+is considered feature-freeze and new feature-requests may be closed within two weeks if no new owner is found.
+[@open-telemetry/javascript-approvers](https://github.com/orgs/open-telemetry/teams/javascript-approvers) may sponsor
+features for unmaintained components. At least one sponsor is needed to lift the feature-freeze for the purpose of
+adding the requested feature. Sponsors are expected to provide reviews for that feature and be responsive on the issue.
+
+Components marked as unmaintained still receive semantic conventions updates where possible.
```

Comment:
We also mentioned that we would allow bug fixes, do you want to clarify this here too?

Reply:
Good point, clarified that in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2272/commits/655740968e0af8276c0554399a6d8e2d8d266e84 :slightly_smiling_face: